utopia-remix/app/components/SharePopup.tsx

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable react/jsx-no-bind */
```

Comment:
You shouldn't need this. If you ever do need to disable rules like this, please only do so for specific lines where possible